typescript/radix-sdk/src/index.ts

```diff
@@ -29,7 +29,7 @@ const NETWORKS = {
 };
 
 export { NetworkId };
-export { RadixIsmTypes } from './utils/types.js';
+export { RadixIsmTypes, RadixTokenTypes } from './utils/types.js';
```

Comment:
I think we shouldn't export this, if we want to have an externally usable `RadixTokenTypes` it should be derived from the already existing `TokenType` so for example it should be something like this:

`export type RadixTokenTypes = TokenType.collateral | TokenType.synthetic` so that `RadixTokenTypes` can be used with `TokenType` but not viceversa